### PR TITLE
GIT manage untracked files

### PR DESCRIPTION
**Pull request by dantleech**
@greg0ire look OK?

### CODE REVIEW

**Review by greg0ire — APPROVED** [review]

File: lib/Adapter/Git/GitFilesystem.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ public function __construct(FilePath $path)
 
     public function fileList(): FileList
     {
-        $gitFiles = $this->exec('ls-files');
+        $gitFiles = array_merge($this->exec('ls-files'), $this->exec('ls-files --others --exclude-standard'));
```
Comment (greg0ire) [inline lib/Adapter/Git/GitFilesystem.php]: I think this is equivalent to `$this->exec('ls-files --cached  --others --exclude-standard')`

**Review by greg0ire — CHANGES_REQUESTED** [review]

File: lib/Adapter/Git/GitFilesystem.php (hunk 1 above)
Comment (dantleech) [reply lib/Adapter/Git/GitFilesystem.php]: Yes indeed, updating.

**Review by greg0ire — APPROVED** [review]
LGTM

**Comment by dantleech** [comment]
:boom:

**Pull request merged**